addons/website/static/src/components/fields/redirect_field.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-name="website.RedirectField" owl="1">
-        <button type="button" class="oe_stat_button btn btn-light w-100 mb-0 py-0 border-0 border-start border-bottom rounded-0 text-start text-nowrap text-capitalize"
-            t-on-click="onClick"
-            t-att-aria-label="info"
-            t-att-title="info">
-            <i class="fa fa-fw o_button_icon fa-globe" t-att-class="this.props.value ? 'text-success' : 'text-danger'"/>
-            <div class="o_stat_info">
-                <span class="o_stat_text">Go to<br/>Website</span>
-            </div>
-        </button>
-    </t>
```

Comment:
Mmh, I would be tempted to let you do that (I am a strong partisan of letting people break the history if it means using code conventions or improving readability) but I would prefer to do this only in dedicated commits (so that when we do use the git history we directly know if we are going to hit a commit that is just moving code around).